### PR TITLE
Sms canary

### PR DESCRIPTION
**Pull request by fleetwoods**

### CODE REVIEW

File: src/main/resources/application.conf [hunk 1]
```diff
@@ -27,13 +27,20 @@ canary {
...
     }
   }
 }
 
+twilio {
+  account_sid = ${TWILIO_ACCOUNT_SID}
+  auth_token = ${TWILIO_AUTH_TOKEN}
+  service_sid = ${CANARY_TWILIO_SERVICE_SID}
+}
+
```
Comment (cb372) [inline src/main/resources/application.conf]: You could add dummy default values for some of this config, just to make it easier to run the app locally. There's no need to provide a lot of the config in environments where the canaries won't run.

e.g.

```
account_sid = "dummy"
account_sid = ${?TWILIO_ACCOUNT_SID}
```

File: src/main/scala/com/ovoenergy/comms/monitor/canary/CanaryCommActor.scala [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      functions.checkDelivery(generatedAt, traceToken) match {
+        case Left(failure) =>
+          logWarn(
+            traceToken,
+            s"Canary comm was delivered but was not as expected - ${failure.toList.mkString(",")}, raising alert")
+          functions.raiseAlert(s"Canary comm was delivered but was not as expected - ${failure.toList.mkString(",")}",
+                               traceToken)
+          dropTheMic()
+        case Right(true) =>
+          logInfo(traceToken, "Canary comm delivered")
```
Comment (cb372) [inline src/main/scala/com/ovoenergy/comms/monitor/canary/CanaryCommActor.scala]: Might be handy to include the word "SMS" in all these log messages?

Comment (cb372) [inline src/main/scala/com/ovoenergy/comms/monitor/canary/CanaryCommActor.scala]: Sorry, ignore that, I thought this was an SMS-specific class. In that case, could pass the channel name as an argument for logging.

**Comment by ovo-comms-circleci** [comment]
This PR was successfully deployed to production by [CircleCI build 193](https://circleci.com/gh/ovotech/comms-monitor-service/193) at 2017-04-13 09:22 BST.

**Pull request merged**